app/views/shared/_header.html.erb

```diff
@@ -32,7 +32,7 @@
           <% if @current_user %>
             <%= link_to "profile", profile_path, :class => active_path_class?(profile_path, '') + " navLink"  %>
           <% else %>
-            <%= link_to Hacktoberfest.ended? ? "profile" : "start hacking", login_path, :class => "button button-small navLink" %>
+            <%= link_to login_cta, login_path, :class => "button button-small navLink" %>
```

Comment:
So a couple of thoughts:

First, about projects: Could we include a callout for when HF has ended, indicating that these are open issues looking for contributions more generally? Right now, between the issues being curated for HF, and the "refreshing" of the issues, it might be misleading for users who want to contribute, if they think they are still doing so for HF.

Which leads me to another question – was there group reasoning behind the absence of a more obvious callout on the homepage to indicate that HF is over? I'm thinking in particular of the `/login` auth flow – and the fact that it's several steps from the click on the cta to an explanation that HF is over. Personally, I would want to know that sooner, so I'm wondering about the logic behind that decision (which perhaps goes back to last year?)

Reply:
:nod: I definitely agree we need some copy around projects to make it clear that whilst folks can still contribute, contributions no longer count toward Hacktoberfest -- and, we could probably do with some sub-text below "Support open source and pick a limited edition T-shirt or plant a tree." mentioning that Hacktoberfest is over.

We can't really do anything about the number of steps a user needs to take from the login cta to seeing the closed message, as folks that have already signed up still need a way to sign in and view their profiles, so we can only show the registration closed message once a user has authed and we've determined they haven't already registered. Adding some copy on the homepage prior to the user ever using the login cta though would most definitely help.

The changes in this PR were what the team requested, but will go back to them on the two copy change points above and see what they say 👍

Reply:
Sounds good!

Reply:
Copy updated for both :)